frontend/src/app/shared/components/datepicker/wp-date-picker-modal/wp-date-picker-instance.component.ts

```diff
@@ -220,31 +221,38 @@ export class OpWpDatePickerInstanceComponent extends UntilDestroyedMixin impleme
       this.injector,
       '#flatpickr-input',
       this.currentDates(),
-      {
-        mode: this.dateMode,
-        showMonths: this.deviceService.isMobile ? 1 : 2,
-        inline: true,
-        onReady: (_date, _datestr, instance) => {
-          instance.calendarContainer.classList.add('op-datepicker-modal--flatpickr-instance');
-
-          this.ensureHoveredSelection(instance.calendarContainer);
-        },
-        onChange: this.onFlatpickrChange.bind(this),
-        // eslint-disable-next-line @typescript-eslint/no-misused-promises
-        onDayCreate: async (dObj:Date[], dStr:string, fp:flatpickr.Instance, dayElem:DayElement) => {
-          onDayCreate(
-            dayElem,
-            this.ignoreNonWorkingDays,
-            await this.datePickerInstance?.isNonWorkingDay(dayElem.dateObj),
-            this.isDayDisabled(dayElem),
-          );
-        },
-      },
+      this.datePickerOptions(),
       // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
       this.flatpickrTarget.nativeElement,
     );
   }
 
+  private datePickerOptions() {
+    const options = {
+      mode: this.dateMode,
+      showMonths: this.deviceService.isMobile ? 1 : 2,
+      inline: true,
+      onReady: (_date, _datestr, instance) => {
+        instance.calendarContainer.classList.add('op-datepicker-modal--flatpickr-instance');
+
+        this.ensureHoveredSelection(instance.calendarContainer);
+      },
+      onChange: this.onFlatpickrChange.bind(this),
+      // eslint-disable-next-line @typescript-eslint/no-misused-promises
+      onDayCreate: async (dObj:Date[], dStr:string, fp:flatpickr.Instance, dayElem:DayElement) => {
+        onDayCreate(
+          dayElem,
+          this.ignoreNonWorkingDays,
+          await this.datePickerInstance?.isNonWorkingDay(dayElem.dateObj),
+          this.isDayDisabled(dayElem),
+        );
+      },
+      minDate: this.minDate,
```

Comment:
Do you know if it is possible to have a more explicit visualization for the minDate?. What we currently have left me wondering a couple of times as to why I couldn't click on a certain date.

Reply:
We have a predefined style for disabled days, in order to change it I need to consult the UI team first:

https://github.com/opf/openproject/blob/5dde7f8e02b34ee0ae4fb73ec7f15a02d0e2a6e9/frontend/src/global_styles/content/_datepicker.sass#L31-L35

Reply:
I applied the `--fgColor-disabled` to the disabled days, and it makes them more easy to spot.
<img width="653" alt="image" src="https://github.com/user-attachments/assets/b15cf874-fa80-4fee-9fee-794567b1100f" />

Dark mode
<img width="655" alt="image" src="https://github.com/user-attachments/assets/a2c47e21-bdd2-46ab-87e7-b6267e2bc65f" />